translate/convert/xliff2po.py

```diff
@@ -67,11 +69,6 @@ def converttransunit(self, transunit):
 
     def convertstore(self, inputfile, duplicatestyle="msgctxt"):
         """Converts a .xliff file to .po format"""
-        # XXX: The inputfile is converted to string because Pootle supplies
-        # XXX: a PootleFile object as input which cannot be sent to PoXliffFile
-        # XXX: The better way would be to have a consistent conversion API.
-        if not isinstance(inputfile, (file, wStringIO.StringIO)):
-            inputfile = str(inputfile)
```

Comment:
Pootle tests are not as extensive as TTK ones and the use mostly PO files, so I don't think testing on Pootle will help.

I see you are replacing `StringIO` with `BytesIO` in other places, so perhaps it is ok to keep this hack with that minor change.